tsconfig.json

```diff
@@ -10,7 +10,7 @@
     "sourceMap": true,
     "allowSyntheticDefaultImports": true,
     "esModuleInterop": true,
-    "resolveJsonModule": false,
+    "resolveJsonModule": true,
```

Comment:
revert

Reply:
How should I handle this I get an error when it is false.

Error:-
![image](https://user-images.githubusercontent.com/76607510/147909421-218ab32f-8f41-4f3c-8f91-8b3f4e1b1932.png)